README.md

```diff
@@ -26,3 +26,13 @@ Documentation for this codebase lives at [docs.calitp.org/data-infra](https://do
 This repository uses black and pre-commit hooks to format code. To install locally, run
 
 `pip install pre-commit` & `pre-commit install` in the root of the repo.
+
+## Configuration via Environment Variables
+Generally we try to configure things via environment variables. In the Kubernetes
+world, these get configured via [Kustomize overlays](./kubernetes/overlays/).
```

Comment:
This link / path doesn't seem to exist (`/kubernetes/overlays/`)? 

Also, sorry for belaboring this point / for what I am presumably missing, but where does this actually get set for the v3 RT archiver? Like I don't see a Kustomize file in this PR for the archiver. Would that happen in a subsequent PR?

Reply:
Ah yeah let me link to the actual file.